src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int sideR;
+    private int height;
+
+    public RightTriangle(int sideR, int height, String color) {
+        super(color);
+        this.sideR = sideR;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+    }
```

Comment:
The draw() method is empty. Consider implementing a way to display the figure's properties, such as type, dimensions, and color.